src/Symfony/Bundle/DebugBundle/DependencyInjection/DebugExtension.php

```diff
@@ -42,6 +43,11 @@ public function load(array $configs, ContainerBuilder $container)
             ->addMethodCall('setMinDepth', array($config['min_depth']))
             ->addMethodCall('setMaxString', array($config['max_string_length']));
 
+        if (class_exists(HtmlDumper::class) && method_exists(HtmlDumper::class, 'setTheme')) {
```

Comment:
`if ('dark' !== $config['theme'] && method_exists(HtmlDumper::class, 'setTheme')) {`

Reply:
The `'dark' !== $config['theme']` will not launch an undefinex index notice with var dumper 4.1?

In that case, the Configuration class will not initialize the $config['theme'] index because 'setTheme' doesn't exists.

it wouldn't be better to set something like:

`if (isset($config['theme'] && 'dark' !== $config['theme'] && method_exists(HtmlDumper::class, 'setTheme')) {`

Or maybe leave it just as:
`if (method_exists(HtmlDumper::class, 'setTheme')) {`
as it appears in the Configuration class.

Reply:
Correct. `if (method_exists(HtmlDumper::class, 'setTheme') && 'dark' !== $config['theme']) {` then!